notebooks/Airline Delays - Data Processing Pipeline.py

```diff
@@ -658,6 +677,38 @@ def chain_delay_analysis (dep_time_diff_one_flight_before, dep_time_diff_two_fli
 
 # COMMAND ----------
 
+# Index label
+labelIndexer = StringIndexer(inputCol="dep_del15", outputCol="label")
+
+train_GBT = labelIndexer.fit(train_GBT).transform(train_GBT)
+validation_GBT = labelIndexer.fit(validation_GBT).transform(validation_GBT)
```

Comment:
train_GBT, validation_GBT, and test_GBT are never defined.

Reply:
Fixed in next commit